src/extension.ts

```diff
@@ -186,25 +184,80 @@ export async function activate(ctx: vscode.ExtensionContext) {
           break;
 
         case Options.never:
-          extensionConfigurationSet(configureOnOpenKey, false, vscode.ConfigurationTarget.Workspace);
+          extensionConfigurationSet(SettingsKey.ConfigureOnOpen, false, vscode.ConfigurationTarget.Workspace);
```

Comment:
I would like the casing of `ConfigureOnOpen` to match `configureOnOpen`. Similar for the other keys